docs/extend/legacy_plugins.md

```diff
@@ -88,6 +88,7 @@ Plugin
  Plugin                                                       | Description
 ------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 [Casbin AuthZ Plugin](https://github.com/casbin/casbin-authz-plugin) | An authorization plugin based on [Casbin](https://github.com/casbin/casbin), which supports access control models like ACL, RBAC, ABAC. The access control model can be customized. The policy can be persisted into file or DB.
+[HBM plugin](https://github.com/kassisol/hbm)                | An authorization plugin.
```

Comment:
Could you add a bit more information to the description? I see there's some information about the plugin in the readme; https://github.com/kassisol/hbm (also mentions that `HBM` stands for "Harbormaster", etc)